src/tools/ceph_objectstore_tool.cc

```diff
@@ -3438,6 +3441,9 @@ int main(int argc, char **argv)
       "Threshold (in seconds) to consider omap listing slow (for op=list-slow-omap)")
     ("dump-data-dir", po::value<string>(&dump_data_dir),
      "Directory to dump object data (for op=dump-export)")
+    ("log-file,l", po::value<string>(&log_file), "log file")
+    ("log-level", po::value<int>(&log_level), "log level (30=most, 20=lots, 10=some, 1=little)")
+    ("conf,c", po::value<string>(&conf_file), "config file")
```

Comment:
I was able to pass "--conf=my-ceph.conf" to regular COT. This is redundant.

Reply:
Agree.